tests/test_traits.py

```diff
@@ -78,10 +82,10 @@ def test_has_trait_object():
     """
     Test the `has_trait` `Operation` method on a simple operation definition.
     """
-    assert TestOp.has_trait(LargerOperandTrait())
-    assert not TestOp.has_trait(LargerResultTrait())
-    assert not TestOp.has_trait(BitwidthSumLessThanTrait(0))
-    assert TestOp.has_trait(BitwidthSumLessThanTrait(64))
+    assert TestOp.has_trait(LargerOperandTrait)
+    assert not TestOp.has_trait(LargerResultTrait)
+    assert not TestOp.has_trait(BitwidthSumLessThanTrait, 0)
+    assert TestOp.has_trait(BitwidthSumLessThanTrait, 64)
```

Comment:
This makes it harder to check that two ops have the same trait, right? You can no longer to `op.has_trait(some_trait_var)`, instead you need to have a tuple `(type[OpTrait], Any)`. Not sure if this is the best way?

Reply:
You can get this with `op.get_traits_of_type(type(some_trait_var))` directly!